zkevm-circuits/src/evm_circuit.rs

```diff
@@ -1296,4 +1302,87 @@ mod test {
                 .assign(&mut layouter, &self.execution_steps)
         }
     }
+
+    // To run this benchmark, comment the `ignore` flag and run the following
+    // command:
+    // `RUSTFLAGS='-C target-cpu=native' cargo test --profile bench
+    // bench_evm_circuit_prover -- --nocapture`
+    #[ignore]
+    #[test]
+    fn bench_evm_circuit_prover() {
+        const DEGREE: u32 = 22;
+        const EXECUTION_STEPS_ROWS_MAX: usize = 1 << (DEGREE - 2);
+        const OPERATIONS_ROWS_MAX: usize = 1 << (DEGREE - 2);
+
+        let mut execution_steps = vec![];
+        for _ in 0..EXECUTION_STEPS_ROWS_MAX - 1 {
+            let execution_step = ExecutionStep {
+                opcode: OpcodeId::ADD,
+                case: Case::Success,
+                values: vec![BigUint::new(vec![0])],
+            };
+            execution_steps.push(execution_step);
+        }
+
+        let mut operations = vec![];
+        for i in 0..OPERATIONS_ROWS_MAX - 1 {
+            let operation = Operation::<Fp> {
+                gc: i as usize,
+                target: Target::Memory,
+                is_write: true,
+                values: [Fp::zero(); 4],
+            };
+            operations.push(operation);
+        }
```

Comment:
Last time I ran the benches with that, after the PublicParameters were generated, this witness caused the proof generation process to panic. 

Have you checked that this doesn't happen?